sys/netinet/cc/cc_chd.c

```diff
@@ -85,8 +86,8 @@
  */
 #define	CC_CHD_DELAY	0x02000000
 
-/* Largest possible number returned by random(). */
-#define	RANDOM_MAX	INT_MAX
+/* Largest possible number returned by prng32(). */
+#define	RANDOM_MAX	RANDOM_MAX
```

Comment:
Cool define! ;)

Reply:
Today is not my day! Thank you for being patient with me.